lib/plugins/plugins.js

```diff
@@ -219,12 +221,15 @@ function prepareWordForPackageNameDisplay (word) {
 }
 
 function prepareName (name) {
+  if (name === 'x-govuk') {
```

Comment:
It's worth making the names clearer even if we're not including it in the "available" list because it appears once people have installed.  It's a small enough change that I'd rather keep it in here but I can raise it as a separate PR if needed.